lib/Runtime/Library/JavascriptLibrary.h

```diff
@@ -892,8 +893,10 @@ namespace Js
 
         void SetCrossSiteForSharedFunctionType(JavascriptFunction * function);
 
+        bool IsPRNGSeeded() { return isPRNGSeeded; }
```

Comment:
I wasn't in the discussion over the coding style guidelines, but I would guess we don't recommend using const this way because we don't use const objects generally.  This usage of const is only useful in practice if you're all in on making a clean const library API.

For small functions like these it adds little value.  The class in question is never used through a const pointer afaik, and the method itself is so trivial that making it const will not protect us from making any mistakes in its implementation.